.github/workflows/super-linter.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           # that .golangci.yml is available to the golangci-lint linter
           # https://github.com/github/super-linter/blob/main/docs/using-rules-files.md
           LINTER_RULES_PATH: /
-          VALIDATE_CLANG_FORMAT: true
+          #VALIDATE_CLANG_FORMAT: false # remove clang-format
```

Comment:
This is a weird way to do it. Why not just set it to "false"? Also I think we can get rid of the comment, it's pretty clear that you're not validating with clang format if you set it to false.

Reply:
The super-linter has a strange way of setting these values and you can not have mixture of true and false values.
https://github.com/marketplace/actions/super-linter#environment-variables
I can remove the comment.

Reply:
Weird!